CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 Bugfixes:
 - Fix warnings regarding non-existent System.ComponentModel.TypeConverter NuGet package by updating minimum Castle Core version to 4.1.0 (#321)
 - Fix disposal of faulted WCF client channels (@jberezanski, #322)
+- Ignore minor/patch level version for AssemblyVersionAttribute as this creates binding errors for downstream libraries (@fir3pho3nixx, Core/#288)
+- Fixing master build to use Castle.Core(loggers et al) 4.1.1 (@fir3pho3nixx, #321, #326)
+- Removing readme.txt (@fir3pho3nixx, #305)
```

Comment:
I wouldn't bother mentioning this in the changelog, the changelog should be a summary of the work done but hidden things like refactoring code or build scripts aren't important to a user.

Can you also rebase this PR on to master, it has some of your changes that are already merged.

Reply:
All done.